File: 프로그래머스/lv0/120854. 배열 원소의 길이/배열 원소의 길이.java
class Solution {
    public int[] solution(String[] strlist) {
        int[] answer = new int[strlist.length];
        // answer의 길이는 strlist의 길이와 동일하게 설정한다
        
        for(int i = 0; i < strlist.length; i++){
            answer[i] = strlist[i].length();
        } // answer의 원소에 strlist의 원소별 길이를 넣어준다
        return answer;
    }
}

File: 프로그래머스/Lv.0/120854. 배열 원소의 길이/배열 원소의 길이.js
function solution(strlist) {
    const answer = new Array(strlist.length);
    // answer의 길이는 strlist의 길이와 동일하게 설정한다
    
    for (let i = 0; i < strlist.length; i++) {
        answer[i] = strlist[i].length;
    } // answer의 원소에 strlist의 원소별 길이를 넣어준다
    return answer;
}
